easybuild/easyconfigs/a/awscli/awscli-1.19.53-GCCcore-10.2.0-Python-3.8.6.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'PythonBundle'
+
+name = 'awscli'
+version = '1.19.53'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('colorama', '0.4.3', {
+        'checksums': ['e96da0d330793e2cb9485e9ddfd918d456036c7149416295932478192f4436a1'],
+    }),
+    ('s3transfer', '0.3.3', {
+        'checksums': ['921a37e2aefc64145e7b73d50c71bb4f26f46e4c9f414dc648c6245ff92cf7db'],
+    }),
+    ('rsa', '4.4.1', {
+        'checksums': ['efaf0c32afee1c136e5cd2e7ceecf2dfc65dac00fb812a1b3b8b72f6fea38dbb'],
+    }),
+    ('docutils', '0.15', {
```

Comment:
Version 0.16 is part of Python 3.8.6 already. Do you need to intentionally downgrade it?

Reply:
Yes, setup.py has a lot of ==